bin/tx-push-www

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+/*
+Push www strings to Transifex, using scratch-l10n's tx-push-src.js script
+
+1. for every file like src/views/.../l10n.json, call tx-push-src.js
+2. also call tx-push-src.js on the general file, src/l10n.json
+
+The format of tx-push-src.js's arguments is:
+$ tx-push-src.js tx-project tx-resource english-json-file
+So, for example, src/views/parents/l10n.json will use:
+$ tx-push-src.js scratch-website parents-l10njson src/views/parents/l10n.json
+
+*/
+
+const glob = require('glob');
+const execSync = require('child_process').execSync;
+
+// determine if this is a dry run, or if we're really pushing to transifex
+let execute = false;
+const args = process.argv.slice(2);
+if (args[0] === '--execute') {
+    process.stdout.write('pushing to transifex...\n');
+    execute = true;
+} else {
+    process.stdout.write('Dry run: pass "execute" as a parameter to add --execute switch to commands\n');
```

Comment:
One little nit, the message should be something like: `'Dry run: pass "--execute" as a parameter to execute push files to transifex\n'`

Reply:
Changed!